Test/DependencyCollector/FunctionalTests/FuncTest/IIS/Iis.cs

```diff
@@ -60,7 +59,9 @@ public static void Start()
         public static void Stop()
         {
             Process process = new Process { StartInfo = StopIis };
```

Comment:
So StartIis was removed and only StartW3Svc remains in Start but Stop only has StopIis, should there be a StopW3Svc?